### ISSUE TITLE
Too long files are not flagged if they end on `///`

### ISSUE THREAD

**Issue by kbjarkefur** [worldbank/stata-linter#55]
Lines that are too long but ends on a `///` is not flagged as too long. For example, the line below is too long before the `///` but is not flagged as too long
```	
iebaltab weight price   , grpvar(tmt) pairoutput("p") ftestoutput("p")  savecsv("${ietoolkit}/run/output/iebaltab/iebt_csv") savexlsx("${ietoolkit}/run/output/iebaltab/iebt_xlsx") /// 
replace
```
